server/index.js

```diff
@@ -1,6 +1,24 @@
-import { app } from './app.js';
-import { PORT } from './utils/config.js';
+import { app } from "./app.js";
+import { PORT } from "./utils/config.js";
+
+app.post("/assignments", async (req, res) => {
```

Comment:
please delete both functions, or move them to controllers/assignments.js - index.js should only run server